Xamarin.PropertyEditing.Mac/Controls/Custom/ResourceOutlineViewDelegate.cs

```diff
@@ -34,10 +34,9 @@ public override NSView GetView (NSOutlineView outlineView, NSTableColumn tableCo
 
 				case ResourceOutlineView.ResourceNameColId:
 				default:
-					var utf = (UnfocusableTextField)outlineView.MakeView (labelIdentifier, this);
+					var utf = (VerticallyCentredUnfocusableTextField)outlineView.MakeView (labelIdentifier, this);
```

Comment:
Nit: I think we usually standardize on US spelling, which would be `VerticallyCenteredUnfocusableTextField` here.